File: benchmarks/fib30/fib30.rs
fn fib(n: u64) -> u64 {
    if n <= 1 {
        return n;
    }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    let n: u64 = 30;
    let result = fib(n);
    println!("{}", result);
}

File: benchmarks/fib30/fib30.dart
int fib(int n) {
  if (n <= 1) {
    return n;
  }
  return fib(n - 1) + fib(n - 2);
}

void main() {
  int n = 30;
  int result = fib(n);
  print(result);
}
